_data/components/overviewchart.yml

```diff
@@ -0,0 +1,72 @@
+directive : "o-chart"
+
+inheritedAttributes: [{
+  component: "OServiceBaseComponent",
+  path: "charts/overview",
+  attributes: ["attr", "service", "service-type", "entity", "query-fallback-function", "query-on-init", "query-on-bind", "pageable", "columns", "keys", "parent-keys", "static-data", "query-method", "paginated-query-method", "query-rows", "insert-method", "update-method", "delete-method", "store-state", "query-with-null-parent-keys"]
```

Comment:
There is some parameters that don't make sense in the chart component like paginated-query-method, insert-method, update-method, delete-method